test/unit/src/common/components/select-repository-row/t_select-repository-row.js

```diff
@@ -79,10 +79,21 @@ describe('The SelectRepositoryRow component', () => {
 
       context('and the disabled prop is "true"', () => {
         beforeEach(() => {
-          props.disabled = true;
+          props.isEnabled = true;
         });
 
-        xit('should contain a disabled checkbox', () => {
+        it('should contain a disabled checkbox', () => {
+          const component = shallow(<SelectRepositoryRow { ...props } />);
+          expect(component.find('input[disabled=true]').length).toBe(1);
+        });
+      });
+
+      context('and the reposistory is fetching', () => {
```

Comment:
Agreed about a bit confusing naming.

Tests (and contexts) should be independent - therefore quite understandable on case by case basis.
Contexts that start with `and ...` suggest dependancy on something that happened before that. If there is a dependency on previous tests - then the tests are not well implemented, if there is no dependency, we shouldn't suggest it in naming.